distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-updating.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="dist-tracing-jaeger-updating"]
+= Updating distributed tracing platform Jaeger
+include::_attributes/common-attributes.adoc[]
+:context: dist-tracing-jaeger-updating
+
+toc::[]
+
+Operator Lifecycle Manager (OLM) controls the installation, upgrade, and role-based access control (RBAC) of Operators in a cluster. The OLM runs by default in {product-title}.
+OLM queries for available Operators as well as upgrades for installed Operators.
+For more information about how {product-title} handles upgrades, see the xref:../../operators/understanding/olm/olm-understanding-olm.adoc#olm-understanding-olm[Operator Lifecycle Manager] documentation.
+
+During an update, the {DTProductName} Operators upgrade the managed {DTShortName} instances to the version associated with the Operator. Whenever a new version of the {JaegerName} Operator is installed, all the {JaegerShortName} application instances managed by the Operator are upgraded to the Operator's version. For example, after upgrading the Operator from 1.10 installed to 1.11, the Operator scans for running {JaegerShortName} instances and upgrades them to 1.11 as well.
+
+For specific instructions on how to update the OpenShift Elasticsearch Operator, see xref:../../logging/cluster-logging-upgrading.adoc#cluster-logging-upgrading_cluster-logging-upgrading[Updating OpenShift Logging].
```

Comment:
```suggestion
```

That part can be removed. All the stuff on top, should be valid for almost every operator.

Reply:
@frzifus, thank you. 

This is the file for Jaeger, so I assume we still keep this part for Jaeger, correct?